.gitignore

```diff
@@ -45,4 +45,5 @@ app.*.map.json
 
 # Generated files
 *.g.dart
-*.freezed.dart
\ No newline at end of file
+*.freezed.dart
+.vscode/launch.json
```

Comment:
Added to ignore my launch profiles. Hope it's okay.

Reply:
Sure, thanks for adding that!